### ISSUE TITLE
Tiny [FR] Show version number in the window title ,e.g.,  "Ultimaker Cura 3.2"

### ISSUE THREAD

**Issue by filipgoc** [Ultimaker/Cura#3295]
**Current State**
![image](https://user-images.githubusercontent.com/16858474/36127969-2f167798-1060-11e8-8f80-82a0c5a996ae.png)

**Desired**

 "Ultimaker Cura 3.2"

**Rationale**

For those of us who run multiple versions of Cura, which should be anyone who is testing any beta for example, it is sometimes confusing to know which version we have currently opened. Sometimes I have multiple opened when I'm comparing behavior, too... 

My current workaround is to edit the graphics files to get instant version recognition, but that feels hackish of course. 

![image](https://user-images.githubusercontent.com/16858474/36128015-6d258128-1060-11e8-84a1-b00936673206.png)
![image](https://user-images.githubusercontent.com/16858474/36128046-8a284a12-1060-11e8-89f8-fe8098321cf9.png)

**Comment by ianpaschal** [comment]
I don't know of a single Mac application with this behavior, and I don't think we would add it quite like you describe. A compromise, though, might be to include version numbers on beta versions instead of stable releases.

**Comment by jackha** [comment]
I like the graphics ^^

**Comment by ianpaschal** [comment]
@LukeChen-Ultimaker

**Comment by filipgoc** [comment]
Yeah, maybe it's not worth it. For the beta versions, it could be worth it. No biggie.

**Comment by fieldOfView** [comment]
I have created a plugin for you:
https://github.com/fieldOfView/VersionInTitlebarPlugin

Easiest way to install this is to download the curaplugin here:
http://files.fieldofview.com/temp/VersionInTitlebarPlugin.curaplugin
Drop it in Cura and restart Cura.

**Comment by filipgoc** [comment]
You're such a beast @fieldOfView !
